legacy.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"),
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const h3v4 = require('./dist/h3-js');
+const mapping = require('./lib/legacy-mapping');
+
+module.exports = Object.keys(mapping).reduce((map, key) => {
+    map[key] = h3v4[mapping[key]];
+    return map;
+}, {});
+
+// {
```

Comment:
this comment block is presumably unneeded?

Reply:
Correct, will remove